Sources/Fluent/Database/Transactable.swift

```diff
@@ -11,12 +11,12 @@ extension Database: Transactable {
     /// be rolled back.
     ///
     /// note: the underlying driver must support transactions
-    public func transaction(_ closure: (Connection) throws -> ()) throws {
+    public func transaction<R>(_ closure: (Connection) throws -> R) throws -> R {
```

Comment:
No worries - thanks for merging it in. Will do the necessary changes in `mysql-driver` now.